ebl-issuance/src/main/java/org/dcsa/conformance/standards/eblissuance/party/EblIssuanceCarrier.java

```diff
@@ -190,6 +193,34 @@ private void sendIssuanceRequest(JsonNode actionPrompt) {
             .formatted(isCorrect ? "correct" : "incorrect", tdr, eblStatesByTdr.get(tdr)));
   }
 
+  private ObjectNode getSupportingDocumentObject(){
+    var document = generateDocument();
```

Comment:
In my view, the document should be unique between each scenario. Otherwise, we could be caught by a business rule where the platform flags the issuance because it saw the PDF for an eBL with different content issued to a different party and now the PDF is being re-used.

Therefore, a static PDF does not seem like it would work. If we had a PDF template where the CTK could inject a single field or value somehow, then we could change to using PDFs.

Reply:
I have modified the logic and pushed the latest changes. I have added a pdf file(the dev portal home pdf ) and modified the existing generateDocument() method to add UUID to it. Please let me know if anything needs to be changed here @nt-gt @gj0dcsa.

Reply:
Seems fine to me. I will leave George to do the final approval.